gateways/core/artifact/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM centos:7
```

Comment:
any reason why you are using this image? can we not build from `scratch`

Reply:
Definitely. I changed it to centos so I could interact with the pod through shell for testing. I've planned to revert it to scratch.